.github/workflows/build.yml

```diff
@@ -4,6 +4,10 @@ on:
     branches:
       - develop
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
we can add this magic cancel to lint proto...all workflow right?

Reply:
yes